src/main/java/org/dcsa/ebl/controller/unofficial/ChangeDocumentStatusController.java

```diff
@@ -0,0 +1,71 @@
+package org.dcsa.ebl.controller.unofficial;
+
+import lombok.RequiredArgsConstructor;
+import org.dcsa.core.events.model.enums.ShipmentEventTypeCode;
+import org.dcsa.core.events.repository.BookingRepository;
+import org.dcsa.core.events.repository.TransportDocumentRepository;
+import org.dcsa.core.exception.ConcreteRequestErrorMessageException;
+import org.dcsa.ebl.repository.ShippingInstructionRepository;
+import org.dcsa.ebl.service.TransportDocumentService;
+import org.springframework.http.MediaType;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import reactor.core.publisher.Mono;
+
+import java.util.UUID;
+
+@RestController
```

Comment:
If you add a @Profile annotation, you can make it only active with certain profiles, like "dev" or "test"

Reply:
Good idea, but might be